pkg/operator/credentialsrequest/credentialsrequest_controller.go

```diff
@@ -71,6 +71,8 @@ const (
 	credentialsRequestInfraMismatch = "InfrastructureMismatch"
 )
 
+var defaultRequeueTime = time.Hour + time.Minute*10
```

Comment:
An hour is conceptually easier to explain to people, is the 10 minute buying us anything? I could see a random jitter like we do in hive but really not a big concern for the scale CCO works at.

Reply:
Also please add a comment on why we're doing a requeue instead of just waiting for events.

Reply:
Since we explicitly don't re-reconcile more than once every hour, I was just thinking if there is a small difference between when the object is requeued and the lastSync values (where it requeues just before 1 hour so the lastSync is still less than 1 hour ago), then we could effectively slip out to reconciling to every 2 hours. Not a huge deal, but it is double the intended reconcile rate.